src/settings-ui/Microsoft.PowerToys.Settings.UI/loc/nl/src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw.lcl

```diff
@@ -3286,8 +3295,8 @@
       <Item ItemId=";PowerRename_Toggle_EnableOnExtendedContextMenu.Header" ItemType="0;.resx" PsrId="211" Leaf="true">
         <Str Cat="Text">
           <Val><![CDATA[Appear only in extended context menu]]></Val>
-          <Tgt Cat="Text" Stat="Update" Orig="New">
-            <Val><![CDATA[Weergeven in uitgebreide contextmenu's]]></Val>
+          <Tgt Cat="Text" Stat="Loc" Orig="New">
+            <Val><![CDATA[Alleen weergeven in uitgebreide contextmenu's]]></Val>
```

Comment:
Don't know where the "-s" came from, but this is all single, not plural. Please correct.

Reply:
Thank you for the feedback. Could you please create an issue for this and we'll bring it up with the loc team. Currently it's an external tool that creates these.